### ISSUE TITLE
File Download restart

### ISSUE THREAD

**Issue by rthangam** [axios/axios#5963]
I discovered that file downloads are not supported. Is there any out-of-the-box solution available, or is it simply not possible? Currently, I'm using the 'abort' option to cancel downloads. I'm wondering if restarting download from an aborted file could serve as a way to resume file downloads?

_Originally posted by @rthangam in https://github.com/axios/axios/discussions/5962_

**Comment by promiseer** [comment]
could you be more specific with the issue , add some more descriptions such as images/video.

**Comment by rthangam** [comment]
Let's say i am trying to download files f1, f2 and f3 in parallel. It works fine and now i am using the abort option to pause the download it works fine. Now when i want to try restart for the paused downloads i don't see any option. My question is: If it is not supported by Axios is there a way to do it in react js. I am using it along with react js. In React js i can't split the files into chunks and access filesystem.

**Comment by rthangam** [comment]
This issue is same as https://github.com/axios/axios/issues/5995

**Comment by rahulbansod519** [comment]
Implement the handleResumeDownload function to send a new request to the server with the appropriate Range header to request the remaining part of the file. Update the state to indicate that the download is no longer paused.

`const handleResumeDownload = (download) => {
  // Create a new Axios request with a Range header to resume the download
  // Update the state to mark it as no longer paused and update the progress
};
`

**Comment by rthangam** [comment]
Thanks @rahulbansod519 but i am using react js and Axios so i can't access fs module to update file chunks

**Comment by jasonsaayman** [comment]
closing in favour of #5995
